breathe/renderer/sphinxrenderer.py

```diff
@@ -1442,7 +1443,10 @@ def visit_docparblock(self, node) -> List[Node]:
     def visit_docimage(self, node) -> List[Node]:
         """Output docutils image node using name attribute from xml as the uri"""
 
-        path_to_image = self.project_info.sphinx_abs_path_to_file(node.name)
+        path_to_image = node.name
+        if not url_re.match(path_to_image):
+            path_to_image = self.project_info.relative_path_to_xml_file(path_to_image)
```

Comment:
@jdknight It used to call abs path function, but now calls rel path to xml file function. At a glance, this appears to be a mistake, can you confirm?

Reply:
I cannot seem to recall why the change (and it's odd that nothing about it was mentioned in the commit; my apologies).

I do not know if this is a mistake though. Looking at the implementation, [`sphinx_abs_path_to_file`](https://github.com/michaeljones/breathe/blob/c84ac087a8a34cbe33cd1912717173819f69aed3/breathe/project.py#L87) is the same as [`relative_path_to_xml_file`](https://github.com/michaeljones/breathe/blob/c84ac087a8a34cbe33cd1912717173819f69aed3/breathe/project.py#L76), but with an injected path separator at the start (so a srcdir-rooted abspath?). It appears off to me to [build a `nodes.images` with a `uri` with a leading path seperator](https://github.com/michaeljones/breathe/blob/c84ac087a8a34cbe33cd1912717173819f69aed3/breathe/renderer/sphinxrenderer.py#L1450) -- unknown what that means for Windows instances and how various plugins interpret the root location of `uri`. I know a Sphinx extension I help maintain, we've observed various plugins generating `uri` entries with relative paths to the working directory, relative paths on the source/environment directory and absolute paths on the system. By injecting the `/` on Linux systems, one may (incorrectly) interpret it is as an absolute path to the entire system (which would not work in this case). I am not sure off hand what is the "right" value for a `uri` entry; I have not seen any specifics in docutils/Sphinx on what is the "right" way to format these `uri` attributes.

That all being said, I have no problems if we adjust this commit back to using `sphinx_abs_path_to_file` to just focus on resolving the external hosted URI entries. If there is a desire to use `relative_path_to_xml_file`, I could go back and re-test against tinyxml and oglft's documentations against various builders and report back.

Reply:
@vermeeren, I have updated the pull request to fall back to the original `sphinx_abs_path_to_file` implementation. This is have this pull request focus on URL-provided images.

Any attempts to switch from `sphinx_abs_path_to_file ` to `relative_path_to_xml_file` could be done in a separate pull request (which can outline in more details on the specific error cases; if any).